### PR TITLE
sequential compact/closed characterization of P99 US

### LINKED ISSUE #594: Maybe it's worth mentioning that US is equivalent to "every sequentially compact subset is sequentially closed"
### Discussed in https://github.com/orgs/pi-base/discussions/591

<div type='discussions-op-text'>

<sup>Originally posted by **Jianing-Song** March 20, 2024</sup>
This is analogous to KC = "every compact subset is compact".

Proof: Suppose that $X$ is US. Let $C$ be a sequentially compact subset of $X$, then there is no sequence in $C$ converging to a point outside $C$, otherwise this sequence has a subsequence converging to a point in $C$ by sequential compactess, and we get a sequence with two different limits. 

Conversely, suppose that every sequentially compact subset is sequentially closed. Each singleton is sequentially compact, hence sequentially closed, which implies that $X$ is $T_1$. Suppose that there is a sequence $(x_n)$ converging to both $a\neq b$. By taking the open neighborhood $U=X\setminus\{b\}$ of $a$ we see that $x_n\neq b$ for all sufficiently large $n\ge N$, and $(x_n)_{n\ge N}\cup\{a\}$ that is not sequentially closed ($b$ is in its sequential closure).</div>

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #594.

### CODE REVIEW

File: properties/P000099.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ refs:
     name: Minimal properties between T1 and T2.
   - doi: 10.4153/CMB-1966-045-1
     name: Semi-Hausdorff Spaces
+  - mathse: 5011974
+    name: Answer to "Can the US separation axiom be characterized by compact sets?"
 ---
 
 Every convergent sequence has exactly one limit.
+
+Equivalently, every {{P20}} subspace
```
Comment (StevenClontz) [inline properties/P000099.md:21]:
```suggestion
Equivalently, every {P20} subspace
```

**Comment by prabau** [comment]
It is an interesting result that US spaces are those in which every sequentially compact set is sequentially closed.

Did you reread this previous comment of mine https://github.com/pi-base/data/issues/594#issuecomment-2008370289 maybe? I was making the point that when we present multiple equivalent definitions for a property, these various definitions are usually semantically close to each other.  And furthermore, we present these multiple formulations because each one of them is sometimes more useful than the other ones in a particular situation.  But we don't necessarily present as equivalent definitions all statements logically equivalent to a property.  In general, there could be a lot more of these, and we usually don't show them in the property definition file (see e.g. many additional variations for T1 in wikipedia).

Anyway, what I was suggesting in #594 was that Jianing Song's result is not a good result as a definition.  It's semantically at a more complicated level than the basic definition and does not seem right to make it an alternate definition.

However I agree that it is interesting.  That's why I was suggesting to make it a note.  I.e., instead of saying "Equivalently, blah blah", we could say something like

"Note: $X$ is US if and only if all sequentially compact sets in $X$ are sequentially closed. ({{mathse ...}})."

Maybe I am overthinking this.  What was your thinking here?

**Comment by StevenClontz** [comment]
I simply thought it was a neat characterization that nicely mirrors the definition of KC. I don't necessarily view our lists of equivalent characterizations as though they are all natural definitions. And I felt adding @Jianing-Song's observation was a strictly positive contribution vs not. (And I wanted to swing at some of our older issues.)

Please feel free to adjust my wording of the observation if you like.

**Comment by prabau** [comment]
I made some changes that I could not do as a suggestion: Minor rewording to better match the definitions of Strongly KC and KC; and added author names for the reference papers.

Approving, but feel free to further change if you see a problem.

FYI, part of the added result is Theorem 7 in Murdeshwar & Naimpally (not a very good paper btw).

**Review by prabau — APPROVED** [review]

**Pull request merged**